### DISCUSSION TITLE
Windows Path

### DISCUSSION THREAD

**Discussion by BlueCorallite** [edrlab/thorium-reader#2083] [Q&A] [answered]
Hi, I'm just wondering where Thorium stores all our books after we import them (Windows). Thanks!

**Comment by danielweck** [comment]
Hello, Thorium is an Electron app that stores its data in the standard system folder:

https://www.electronjs.org/docs/latest/api/app#appgetpathname

...which is %APPDATA% in Windows file explorer

The full path should be something like:

c:/users/Daniel/appdata/local/EDRLab.Thorium.Reader/publications

(or "roaming" instead of "local")

See issue:

https://github.com/edrlab/thorium-reader-doc/issues/11#issuecomment-1238399480

The documentation does not currently explicitly mention this:

https://thorium.edrlab.org/docs/220_organizing/221_libraries/

**Comment by horus68** [comment, marked as answer]
On Windows 10 Home files are placed here:
C:\Users\xxxxx\AppData\Roaming\EDRLab.ThoriumReader\publications

**Comment by BlueCorallite** [comment]
Awesome, thanks for your help!
